archivist/runner.py

```diff
@@ -9,7 +9,7 @@
 from logging import getLogger
 from time import sleep as time_sleep
 from types import GeneratorType
-from typing import TYPE_CHECKING, Any, Callable
```

Comment:
Removed Callable because the pyright checker insisted on incorrect settings in other classes and methods but this proved too difficult to fix